interface/wx/dirctrl.h

```diff
@@ -19,6 +19,10 @@ enum
     wxDIRCTRL_EDIT_LABELS    = 0x0100,
     // Allow multiple selection
     wxDIRCTRL_MULTIPLE       = 0x0200,
+    // Enable right-click menu
+    wxDIRCTRL_RCLICK_MENU               = 0x0400,           // Create an r-click menu, even if the next two options aren't used.
```

Comment:
Minor style point: could you please avoid overlong (much longer than 80 characters) lines? This notably makes reviewing the changes when viewing side-by-side diffs much more pleasant.

In practice, this means putting the comment before the enum item (and wrapping it if necessary).